### PR TITLE
Update ode_utils.py

### PR DESCRIPTION
**Pull request by Faezehabibi**
RK4 added
_sum_combine function changed
RK2 changed in names to be aligned with RK4

### CODE REVIEW

**Comment by Faezehabibi** [comment]
RK4 added
_sum_combine function changed
RK2 changed in names to be aligned with RK4

File: ngclearn/utils/diffeq/ode_utils.py
Comment (ago109) [inline ngclearn/utils/diffeq/ode_utils.py:1]: typo in comment -- change "rk4$" to "rk4"

File: ngclearn/utils/diffeq/ode_utils.py [hunk 1]
```diff
@@ -143,12 +149,60 @@ def step_rk2(t, x, dfx, dt, params, x_scale=1.):
...
+    f_2 = dfx(t1, x1, params)
+    _t, _x = _step_forward(t, x, f_2, dt, x_scale)
     return _t, _x
 
+
+def step_rk4(t, x, dfx, dt, params, x_scale=1.):
+    """
+    Iteratively integrates one step forward via the midpoint method, i.e., a
+    fourth-order Runge-Kutta (RK-4) step.
+    (Note: ngc-learn internally recognizes "rk4$" or this routine)
```
Comment (ago109) [inline ngclearn/utils/diffeq/ode_utils.py:164]: typo in comment for Rk4 doc-string -- change "rk4$" to "rk4"

**Review by ago109 — COMMENTED** [review]
Looks good overall; minor typo in rk4 to fix and make sure that markdown files that you want to commit are included in this pull request

**Pull request closed without merging**